assets/app/features/upgrade/steps.factory.spec.js

```diff
@@ -50,12 +50,292 @@ describe('Stepes Factory', function () {
             active: false,
             finished: false
         }
-    ];
+        ],
+        initialStatusData = {
+            current_step: 'upgrade_prechecks',
+            substep: null,
+            current_node: null,
+            steps: {
+                upgrade_prechecks: {
```

Comment:
Follow up card: https://trello.com/c/ZOJZdSuH/2120-research-options-for-efficient-unit-testing-of-responses-from-status-api